src/platform/forms-system/src/js/patterns/array-builder/ArrayBuilderSummaryPage.jsx

```diff
@@ -637,7 +665,16 @@ export default function ArrayBuilderSummaryPage(arrayBuilderOptions) {
       });
 
       if (isValid) {
+        // NOTE: Blocking submission using duplicateChecks.allowDuplicates is
+        // not enabled in MVP because we need to consider UX of the modal first
+        // if (
+        //   duplicateChecks?.allowDuplicates === false &&
+        //   duplicateCheckResult.hasDuplicate
+        // ) {
+        //   scrollAndFocus('va-card:has(.array-builder-duplicate-alert)');
+        // } else {
```

Comment:
I opted to not include the `allowDuplicates` setting (set to `false`) because our team didn't need this feature and the UX wasn't fully thought out.